bundles/org.eclipse.compare/compare/org/eclipse/compare/internal/ComparePreferencePage.java

```diff
@@ -256,15 +256,16 @@ protected Control createContents(Composite parent) {
 		fOverlayStore.load();
 		fOverlayStore.start();
 
-		TabFolder folder= new TabFolder(parent, SWT.NONE);
+		CTabFolder folder= new CTabFolder(parent, SWT.NONE);
 		folder.setLayout(new TabFolderLayout());
 		folder.setLayoutData(new GridData(GridData.FILL_BOTH));
 
-		TabItem item= new TabItem(folder, SWT.NONE);
+		CTabItem item= new CTabItem(folder, SWT.NONE);
+		folder.setSelection(0);
```

Comment:
With https://github.com/eclipse-platform/eclipse.platform.swt/issues/46 fixed, this line would not be needed.